### ISSUE TITLE
Folder modinput/ missing.

### ISSUE THREAD

**Issue by thokroll** [ernw/tr19-badge-firmware#1]
Building the firmware fails with following error:

```
Use make V=1 or set BUILD_VERBOSE in your environment to increase build verbosity.
GEN build/genhdr/qstrdefs.generated.h
<stdin>:97:31: fatal error: modinput/qstrdefs.h: No such file or directory
compilation terminated.
CC ../../py/mpstate.c
CC ../../py/nlr.c
CC ../../py/nlrx86.c
CC ../../py/nlrx64.c
CC ../../py/nlrthumb.c
CC ../../py/nlrxtensa.c
CC ../../py/nlrsetjmp.c
CC ../../py/malloc.c
CC ../../py/gc.c
../../py/gc.c: In function 'gc_sweep':
../../py/gc.c:279:68: error: 'MP_QSTR___del__' undeclared (first use in this function)
                         mp_load_method_maybe(MP_OBJ_FROM_PTR(obj), MP_QSTR___del__, dest);
                                                                    ^
../../py/gc.c:279:68: note: each undeclared identifier is reported only once for each function it appears in
../../py/mkrules.mk:47: recipe for target 'build/py/gc.o' failed
make: *** [build/py/gc.o] Error 1
```

As it seems the folder modinput was not part of the commit.
